item.html

```diff
@@ -412,7 +382,11 @@ <h2 class="ftco-heading-2">Have a Questions?</h2>
 	};
   
 	  firebase.database().ref().update(updates);
```

Comment:
I Think for this part of code we should change the alert and redirect to an asynchronous version either using async/await or with javascript promises